### PR TITLE
[CoSimulationApplication] Minor correction in README documentation

### PR DESCRIPTION
**Pull request by loumalouomega**
**📝 Description**

Minor correction in README documentation regarding `ConvergenceAccelerator` 

**🆕 Changelog**

- [Minor correction in README documentation](https://github.com/KratosMultiphysics/Kratos/commit/e57238305d3433f58982b4beb0a56aea2c74812c)

### CODE REVIEW

File: applications/CoSimulationApplication/README.md [hunk 1]
```diff
@@ -238,7 +238,7 @@ The _CoSimulationApplication_ consists of the following main components (taken f
 - **IO**: Responsible for communicating and data exchange with external solvers/codes
 - **DataTransferOperator**: Transfers data from one discretization to another, e.g. by use of mapping techniques
 - **CouplingOperation**: Tool for customizing coupled simulations
-- **ConvergenceAccelerator**: Accelerating the solution in strongly coupled simulations by use of relaxation techniques
+- **ConvergenceAccelerator**: Accelerating the solution in strongly coupled simulations by use of Jacobiaan-free or Quasi-Newton techniques
```
Comment (loumalouomega) [inline applications/CoSimulationApplication/README.md]:
```suggestion
- **ConvergenceAccelerator**: Accelerating the solution in strongly coupled simulations by use of Jacobian-free or Quasi-Newton techniques
```

File: applications/CoSimulationApplication/README.md [hunk 2]
```diff
@@ -238,7 +238,7 @@ The _CoSimulationApplication_ consists of the following main components (taken f
 - **IO**: Responsible for communicating and data exchange with external solvers/codes
 - **DataTransferOperator**: Transfers data from one discretization to another, e.g. by use of mapping techniques
 - **CouplingOperation**: Tool for customizing coupled simulations
-- **ConvergenceAccelerator**: Accelerating the solution in strongly coupled simulations by use of relaxation techniques
+- **ConvergenceAccelerator**: Accelerating the solution in strongly coupled simulations by use of Jacobian-free or Quasi-Newton techniques
```
Comment (rubenzorrilla) [inline applications/CoSimulationApplication/README.md]:
```suggestion
- **ConvergenceAccelerator**: Accelerating the solution in strongly coupled simulations by use of relaxation or Quasi-Newton techniques
```
As far as I know there are no Jacobian-free techniques implemented. However, there is an Aitken relaxation convergence accelerator.

Comment (loumalouomega) [reply applications/CoSimulationApplication/README.md]: @pooyan-dadvand this is what I suggested

**Review by philbucher — CHANGES_REQUESTED** [review]
Why do you want to change this? IMO relaxation includes all the other stuff and is hence the most generic

**Comment by loumalouomega** [comment]
> Why do you want to change this? IMO relaxation includes all the other stuff and is hence the most generic

Suggestion by @pooyan-dadvand, certainly I am not an expert

**Comment by rubenzorrilla** [comment]
> Why do you want to change this? IMO relaxation includes all the other stuff and is hence the most generic

Indeed, I tend to agree with this observation.

**Comment by loumalouomega** [comment]
> > Why do you want to change this? IMO relaxation includes all the other stuff and is hence the most generic
> 
> Indeed, I tend to agree with this observation.

@pooyan-dadvand

**Comment by loumalouomega** [comment]
> > > Why do you want to change this? IMO relaxation includes all the other stuff and is hence the most generic
> > 
> > 
> > Indeed, I tend to agree with this observation.
> 
> @pooyan-dadvand

Hello @pooyan-dadvand, how do we finally set it?

File: applications/CoSimulationApplication/README.md (hunk 2 above)
Comment (pooyan-dadvand) [reply applications/CoSimulationApplication/README.md]: @rubenzorrilla knows better than me this part. So I would apply his suggestion

Nevertheless, conceptually, I don't understand why we have relaxations under convergence accelerator....

**Review by philbucher — APPROVED** [review]
seems like I have to update my thesis to satisfy @pooyan-dadvand s requests xD

**Pull request merged**